src/megaapi.cpp

```diff
@@ -5706,6 +5711,38 @@ MegaInputStream::~MegaInputStream()
 
 }
 
+MegaApiLock::MegaApiLock(MegaApiImpl* ptr, bool lock) : api(ptr)
```

Comment:
Can we pass-by-ref here and name it `api`, not `ptr`?

Reply:
done :-)